doc/src/conf.py

```diff
@@ -32,7 +32,7 @@
 mathjax_path = 'http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML-full'
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['.templates']
+templates_path = ['_templates']
```

Comment:
I'm just following the naming scheme that's already there for the `_static` directory. There wasn't a `.templates` directory so I don't think this breaks anything.